### PR TITLE
fix: udpate GSOC page

### PR DESCRIPTION
**Pull request by alexgarel**
GSoC is coming !

### CODE REVIEW

File: lang/en/texts/google-summer-of-code.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!-- no side column -->
 
-<h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
+<h1>Open Food Facts - Google Summer of Code (GSOC)</h1>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:3]:
```suggestion
<h1>Open Food Facts - Google Summer of Code (GSOC) 2024</h1>
```

Maybe it's a good practice to make sure the document is updated each year, to appapt to GSoC rule modification

File: lang/en/texts/google-summer-of-code.html [hunk 2]
```diff
@@ -10,76 +10,74 @@ <h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
...
     <li>Enthusiasm for Open Food Facts goals and its community</li>
     <li>Good communication skills (as needed for good team work in an open environment)</li>
-    <li>Automonomy (more an entrepreneur than a trainee)</li>
-    <li>Experience with the tools you will need to complete the project (eg. demonstrated ability in the relevant programming language or environment)</li>
+    <li>Autonomy (more an entrepreneur than a trainee)</li>
+    <li>First experience with the tools you will need to complete the project (eg. demonstrated ability in the relevant programming language or environment)</li>
     <li>Information on how you would approach the project, what time you can put into it and what you think you will be able to achieve over the GSoC period</li>
 </ul>
-<p>Claiming you have those characteristic is easy, but on our side the only way we can judge those is by the actual things you will accomplish during the bounding period. That's why this period is very important.</p>
+<p>The actual things you will accomplish during the bounding period is the best way to show you meet those criterias.</p>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:32]:
```suggestion
<p>The actual things you will accomplish during the bounding period are the best way to demostrate you meet those criteria.</p>
```

File: lang/en/texts/google-summer-of-code.html [hunk 3]
```diff
@@ -10,76 +10,74 @@ <h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
...
+
+<h3>Why you should candidate</h3>
+<p>Open Food Facts is, first of all, a community. We try our best to be very welcoming and inclusive.</p>
+<p>Food is a very important topic to everybody on the planet and has a major impact on health and environment. We try to inform and help people make better choices everyday.</p>
+<p>Open Data enables is also precious as it enables a lot of re-usage for different use-case (we know of more than 200 of them). It is a precious tool to researchers (500+ citations).</p>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:37]:
```suggestion
<p>Open Data is also precious as it enables many re-usages for various use cases (more than 200 other projects use it). It is a precious tool to researchers (500+ citations).</p>
```

File: lang/en/texts/google-summer-of-code.html [hunk 4]
```diff
@@ -10,76 +10,74 @@ <h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
...
+<p>Your contribution to this project will be visible by a lot of people.</p>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:38]:
```suggestion
<p>Your contribution to this project will impact a lot of people.</p>
```

File: lang/en/texts/google-summer-of-code.html [hunk 5]
```diff
@@ -10,76 +10,74 @@ <h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
...
-    <li>As you learn, it is also a good idea to propose updates to the documentation.</li>
-    <li>If you have not worked before with Git Branching, we encourage you to visit this web: <a href="https://learngitbranching.js.org/">https://learngitbranching.js.org/</a></li>
-    <li>Look at the <a href="https://github.com/openfoodfacts">list of all the Open Food Facts projects</a></li>
     <li>Read the project's README on the repository, feel free to ask for necessary clarifications</li>
     <li>Lurk on the project's chat channel</li>
-    <li>Look at the recent changes in the project's GitHub repository</li>
-    <li><a href="https://forms.gle/DzcwqUD79u7os9Lv6"><b>Fill our introduction survey</b></a> (it is NOT an application form)</li>
+    <li>Once you're able to run it, try one of our "good first issues" on GitHub</li>
+    <li>Please, go through the available tutorials and <a href="https://stackoverflow.com/help/how-to-ask">direct your questions</a> to the Slack chat, in the most specific channel possible.</li>
+    <li>As you learn, it is also a good idea to propose updates to the documentation (especially on argument you struggled to understand).</li>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:70]:
```suggestion
    <li>As you learn, it is also a good idea to propose updates to the documentation (especially on arguments you struggled to understand).</li>
```

Comment (alexfauquette) [reply lang/en/texts/google-summer-of-code.html:70]: or
```suggestion
    <li>As you learn, it is also a good idea to propose updates to the documentation (especially on an argument you struggled to understand).</li>
```

File: lang/en/texts/google-summer-of-code.html [hunk 6]
```diff
@@ -10,76 +10,74 @@ <h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
...
+    <li>If you have not worked before with Git Branching, we encourage you to visit this web: <a href="https://learngitbranching.js.org/">https://learngitbranching.js.org/</a></li>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:71]:
```suggestion
    <li>If you have never worked with Git Branching, we encourage you to visit this web: <a href="https://learngitbranching.js.org/">https://learngitbranching.js.org/</a></li>
```

Comment (alexfauquette) [reply lang/en/texts/google-summer-of-code.html:71]:
```suggestion
    <li>If it is your first time using Git Branching, we encourage you to visit this web: <a href="https://learngitbranching.js.org/">https://learngitbranching.js.org/</a></li>
```

File: lang/en/texts/google-summer-of-code.html [hunk 7]
```diff
@@ -10,76 +10,74 @@ <h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
...
 
 
+<h3>Fill out the Application</h3>
+
+<p>Once the application period has opened, you have to submit your application on the Google Summer of Code website. Your application must be written in English. It should contain a detailed description of your project proposal.</p>
 
+<p>Copy our <a href="https://docs.google.com/document/d/1Xb6aOlPxlnvsGLoyROyGk-qnOjFozQ7jqupfbV8H2u0/edit?usp=sharing">Google Document template</a> and make sure you answer all of the questions.</p>
+
+<p>Please be factual and clear in answering these questions. Feel free to add anything else that is relevant for your application. It is never too early to start working on your GSoC application! Note that GSoC positions are very competitive (with about 4 applicants for one position in the past) The key to creating a strong proposal is to propose a manageable and agreed-upon project, make a contribution to the module your proposal is related to, and write an application that clearly demonstrates your knowledge, skills, and enthusiasm.</p>
```
Comment (alexfauquette) [inline lang/en/texts/google-summer-of-code.html:81]:
```suggestion
<p>Please be factual and clear in answering these questions. Feel free to add anything else that is relevant to your application. It is never too early to start working on your GSoC application! Note that GSoC positions are very competitive (with about 4 applicants for one position in the past) The key to creating a strong proposal is to propose a manageable and agreed-upon project, make a contribution to the module your proposal is related to, and write an application that clearly demonstrates your knowledge, skills, and enthusiasm.</p>
```

**Review by alexfauquette — APPROVED** [review]
Mostly grammarly suggestion

I don't think breaking space impacts the layout. To simplify Github suggestion process, I tend to always break space at the end of a sentence such that 1 sentence = 1 line and so reviews can add suggestions at the sentence level

File: lang/en/texts/google-summer-of-code.html (hunk 1 above)
Comment (alexgarel) [reply lang/en/texts/google-summer-of-code.html:3]: Huuum, I was more on the verge of avoiding this page to decay ;-)

Those are most best practice that we decided.

**Pull request merged**